src/box/lua/config/applier/app.lua

```diff
@@ -1,5 +1,11 @@
 local log = require('internal.config.utils.log')
 local utils_file = require('internal.config.utils.file')
+local expression = require('internal.config.utils.expression')
+
+local conditional_vars = {
```

Comment:
Nit: I think that the `conditional_vars` name fits good in `cluster_config.lua` (because it is about conditional sections), but it doesn't fit here. Something like `fail_if_vars` would look more appropriate, I think.

The same for `applier/roles.lua`.

Reply:
ok, renamed.